src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ProjectConfigurationFileChangeDetector.cs

```diff
@@ -74,7 +74,7 @@ public async Task StartAsync(string workspaceDirectory, CancellationToken cancel
 
```

Comment:
Why the changes to this file?

Reply:
If we don't trim `workspaceDirectory` before passing it to `FileSystemWatcher` then when it eventually finds the file it reports the path as something like `C:\some\dir\/deeper/dir/project.razor.json`, while we'll be comparing it against `C:\some\dir/deeper/dir/project.razor.json`, and they won't match due to the extra `\`. I can write a comment about that above this, it certainly deserves documenting.

Reply:
Ah, that makes sense but why the other changes about normalizing the physicalFilePath? It looks like it was already normalizing is why I ask.